src/common/cockpitcloserange.c

```diff
@@ -0,0 +1,92 @@
+#include "cockpitcloserange.h"
```

Comment:
Copyright header please, and #include "config.h" as the first line.

Reply:
Done, and also added the missing `#pragma once`.